.github/vale/config/vocabularies/Backstage/accept.txt

```diff
@@ -513,3 +513,6 @@ zoomable
 zsh
 scrollable
 severities
+Intellij
+intellij
+IntelliJ
```

Comment:
```suggestion
intellij
```

if you have an all-lowercase version, it automatically means "accept any mixed case too"